content/en/docs/tasks/administer-cluster/reconfigure-kubelet.md

```diff
@@ -101,14 +101,14 @@ and debug issues. The compromise, however, is that you must start with knowledge
 of the existing configuration to ensure that you only change the fields you
 intend to change.
 
-Ideally, the Kubelet would be bootstrapped from a file on disk
+Ideally, the kubelet would be bootstrapped from a file on disk
 and you could edit this file (which could also be version-controlled),
-to create the first Kubelet ConfigMap
-(see [Set Kubelet parameters via a config file](/docs/tasks/administer-cluster/kubelet-config-file)),
-Currently, the Kubelet is bootstrapped with **a combination of this file and command-line flags**
+to create the first kubelet ConfigMap
+(see [Set kubelet parameters via a config file](/docs/tasks/administer-cluster/kubelet-config-file)),
```

Comment:
@sftim , would you double check that the sentence starting on line 104 ends somewhere? I am having trouble parsing the content. Thanks.

Reply:
Reworded.